src/DbUtils.php

```diff
@@ -1269,7 +1269,9 @@ public function getTreeValueCompleteName($table, $ID, $withcomment = false, $tra
                 $name = $result['completename'];
             }
 
-            $name = CommonTreeDropdown::sanitizeSeparatorInCompletename($name);
+            if ($name !== null) {
```

Comment:
I wonder if changing the method signature, allowing null and handling it there would not be a better idea.

Reply:
Changing method signature may trigger warning if someone overrided it, but as method is always call on `CommonTreeDropdown` directly, it make no sense to override it.